test/codeu/chat/server/BasicControllerTest.java

```diff
@@ -89,4 +91,39 @@ public void testAddMessage() {
         "Check that the message has a valid reference",
         message == null);
   }
+
+  @Test
+  public void testAllowsNormalChar() {
+
+    final String alphaNumeric = "Ab0";
```

Comment:
Awesome start. You might think about using a few examples of usernames that contain these characters, that way it's a little more obvious what the intended types of names are.